ci/provisioning/post_provision_config_nodes_EL_8.sh

```diff
@@ -55,19 +55,21 @@ install_mofed() {
     stream=false
     gversion="$VERSION_ID"
     if [ "$gversion" == "8" ]; then
-        gversion="8.6"
+        # Mellanox does not have a release for 8.9 yet.
+        gversion="8.8"
         stream=true
      elif [[ $gversion = *.*.* ]]; then
         gversion="${gversion%.*}"
     fi
 
     # Add a repo to install MOFED RPMS
-    repo_url=https://artifactory.dc.hpdd.intel.com/artifactory/mlnx_ofed/"$MLNX_VER_NUM-rhel$gversion"-x86_64/
+    artifactory_base_url="https://artifactory.dc.hpdd.intel.com/artifactory/"
+    mellanox_proxy="${artifactory_base_url}mellanox-proxy/mlnx_ofed/"
+    mellanox_key_url="${artifactory_base_url}mlnx_ofed/RPM-GPG-KEY-Mellanox"
+    rpm --import "$mellanox_key_url"
+    repo_url="$mellanox_proxy$MLNX_VER_NUM/rhel$gversion/x86_64/"
```

Comment:
Since you are modifying all of this, is there any reason this shouldn't be in repo files rather than being added on every CI run (and only for CI runs -- leaving developers without this repo by default)?

Reply:
The problem with putting it in the repo files is that we would have to fix the version to "latest" for that distro and we would have no way to test / pin specific versions.